docs/usage/config-presets.md

```diff
@@ -5,23 +5,23 @@ description: Renovate's support for ESLint-like shareable configs
 
 # Shareable Config Presets
 
-This document describes how to configure your shared presets.
+This page describes how to configure your shared presets.
 Read the [Key concepts, presets](./key-concepts/presets.md) page to learn more about presets in general.
 
-Shareable config presets can only be used with the JSON format, other formats are not supported.
+Shareable config presets must use the JSON format, other formats are not supported.
 
 <!-- prettier-ignore -->
 !!! warning
-    `default.json` is intended for use with presets only!
-    Also, do not use a `renovate.json` file as a preset.
+    Only use `default.json` for your presets.
 
 <!-- prettier-ignore -->
-!!! info
-    We've deprecated the use of a `renovate.json` file for presets as this can cause issues if the repository configuration uses a `renovate.json` file as well.
+!!! warning
+    We've deprecated using a `renovate.json` file for presets, as this causes issues if the repository configuration _also_ uses a `renovate.json` file.
+    If you're using a `renovate.json` file for presets, rename it to `default.json`.
```

Comment:
I think we should explain _what_ a user should do if they're using a `renovate.json` file for their presets. We probably want them to rename their `renovate.json` with the presets?

We also need to avoid confusing our readers who are "extending" from other presets. Maybe it should be worded like this?

```suggestion
    If you're using a `renovate.json` file to share your presets, rename it to `default.json`.
```